src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard.js

```diff
@@ -987,10 +991,22 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
                   Select action
                 </Button>
               }
+              dropdownStyle={{ left: 0, minWidth: 205 }}
             >
-              <DropdownItem>Export to TSV</DropdownItem>
               <DropdownItem
-                style={{ lineHeight: '1.5', cursor: 'pointer', width: 210 }}
+                onClick={() =>
```

Comment:
Add `style={styles.actionMenuItem}` here to make all the dropdownItem consistent